cypress/e2e/list-management/settings-change-annual-review.feature

```diff
@@ -0,0 +1,29 @@
+Feature: List management change annual review date
+
+  Background:
+    Given I am logged in as a "SuperAdmin"
+    And I click the link "Lists"
+    And I click the link "Settings"
+    And I click the link "Change"
+    And I see page with heading "Change annual review start date"
+
+  Scenario: Can change annual review date
+    Given I enter "1" in the "day" input
+    And I enter "2" in the "month" input
+    And I click the "Continue" button
+    And I see page with heading "Confirm new annual review start date"
+    And I click the "Continue" button
+    Then I see the notification text "Annual review date updated successfully"
+    And I should see "1 February" as part of the date
```

Comment:
if you don't write unit tests for this -- you should test multiple different scenarios using a scenario outline. although I'd suggest just wrapping up the date validation and running multiple unit tests on that. 

Otherwise you'd have to keep passing in and changing around the request object.

Reply:
Fair point. I'll put some tests together.